src/components/Transaction.test.js

```diff
@@ -3,12 +3,17 @@ import { render } from '@testing-library/react'
 import Transaction from './Transaction'
 
 describe('Single Transaction', () => {
-  it('shows a paragraph containing a timestamp (date, month, year) and the transaction value', () => {
+  it('Shows a section containing a timestamp (date, month, year) and the transaction value', () => {
     const createdAt = '9.7.2020'
     const newTransaction = '20,95'
-    const { getByText } = render(
-      <Transaction createdAt={createdAt} newTransaction={newTransaction} />
+    const type = 'income'
+    const container = render(
+      <Transaction
+        createdAt={createdAt}
+        newTransaction={newTransaction}
+        type={type}
+      />
     )
-    expect(getByText('9.7.2020 20,95')).toBeInTheDocument()
+    expect(container.firstChild).toMatchSnapshot()
   })
```

Comment:
For cleaner code, it would be great to use here also <Transaction> instead of firstChild or just container?